.gitignore

```diff
@@ -1,3 +1,14 @@
+/bin/console
```

Comment:
Is this change related? If not, then please revert and put it in a separate PR. thanks!

Reply:
If you setup the test environment these files/folders are not ignored. Yes I can split it to an other PR if you like.

Reply:
yes, please 😊

Reply:
@dvesh3 Done